ci/Jenkinsfile.android

```diff
@@ -103,6 +103,7 @@ pipeline {
     }
     stage('Cleanup') {
       steps {
+        cmn.nix.shell 'watchman watch-del $(pwd)'
```

Comment:
Wouldn't it make more sense to just run this in the `clean` target of our `Makefile`?

Reply:
That's a dev-side scenario and outside of the scope of this PR (though we could add it if we're sure of what we want to do there). This is just to make sure we don't leave new watches running in CI.

Reply:
Then at least make it a separate target like `watchman-clean` and call it with `sh 'make watchman-clean'`, othewise you have to add a `script {}` block in the Jenkinsfile.